test/media/ttml_text_parser_unit.js

```diff
@@ -418,6 +418,25 @@ describe('TtmlTextParser', function() {
         '</tt>');
   });
 
+  it('aligns center when textAlign is center', function() {
```

Comment:
Let's merge this test and a previous one into a single it('parses cue alignment from textAlign attribute') with two calls to verifyHelper inside it. (The same way 'disregards empty divs and ps' does 3 calls).